library/system/src/modules/Initrd.rb

```diff
@@ -400,9 +400,9 @@ def Write
         path(".target.bash"),
         Builtins.sformat(
           "/sbin/mkinitrd %1 %2 >> %3 2>&1",
-          param,
-          @additional_parameters,
-          Ops.add(Directory.logdir, "/y2logmkinitrd")
+          param, # cannot escaped due multiple possible params
+          @additional_parameters, # cannot escaped due multiple possible params
+          File.join(Directory.logdir, "y2logmkinitrd")
```

Comment:
shell-escape the result of this? Are we 100% sure `Directory.logdir` can never contain special characters like, say, a blank?

Reply:
it should not, but to be sure, I will add it, so it is consistent with rule "escape or explain why not possible".